### PR TITLE
request token for ocp-qe team

### PR DESCRIPTION
**Pull request by jianlinliu**
Following https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry to get a token for ocp-qe team

### CODE REVIEW

File: clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  namespace: ocp-qe
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+      - secrets
+    verbs:
+      - get
+      - list
```
Comment (petr-muller) [inline clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml:58]: Can you please update this to the current form of the https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry doc?

Individual teams no longer need to define this `secret-reader` role themselves and should just use the provided `pull-secret-namespace-manager` cluster role.

Comment (jianlinliu) [reply clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml]: Thanks for your comments, updated it accordingly.

Comment (alvaroaleman) [reply clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml]: You did not, the rolebinding still references the `secret-reader`

Comment (jianlinliu) [reply clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml]: Oops, my fault, amended it, pls help review again.

File: clusters/app.ci/registry-access/ocp-qe/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+[Document Guide to reqeust service account](https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry)
+
+
+## Generating an Image Pull Credential
+
+First, log in to [the cluster](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/topology/all-namespaces/graph). Then, run:
+
+```sh
+oc get secrets --namespace ocp-qe -o json | jq '.items[] | select(.type=="kubernetes.io/dockercfg") | select(.metadata.annotations["kubernetes.io/service-account.name"]=="image-puller") | .data[".dockercfg"]' --raw-output | base64 --decode | jq
```
Comment (petr-muller) [inline clusters/app.ci/registry-access/ocp-qe/README.md:11]: Note that there is a simpler way documented in https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry

```
oc --namespace my-project registry login --service-account image-puller --registry-config=/tmp/config.json
```

**Review by petr-muller — APPROVED** [review]

**Pull request merged**